code/game/objects/structures/girders.dm

```diff
@@ -41,7 +41,7 @@
 		else if(!anchored)
 			playsound(src.loc, 'sound/items/Ratchet.ogg', 100, 1)
 			to_chat(user, "\blue Now securing the girder")
-			if(get_turf(user, 40))

```

Comment:
It dates back to circa 2012 tg code https://github.com/ParadiseSS13/Paradise/commit/3bd93d5#diff-ac67a04d0e99b8458714248c84118bedR45